strangerstudios/blockville
# Eventblock

### All Submissions:




Added a new package.json file for running npm run start. I updated functions.php to include code that registers for a new custom block called events. I made this code a bit more abstract because it simplifies the act of me registering new blocks. I also added a php and js file for the block. I added a php file for the block because the ourrendercallback function in functions.php takes whatever code I insert in events.php and converts it into a string. This makes personalizing and styling the block a lot easier for me. 

Resolves XXX.

### How to test the changes in this Pull Request:

Go into your local admin and find the block called events. While nothing shows up yet after you insert it, I wanted to make sure everything was up and running before adding any styles.

### Other information:

* [x ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x ] Have you successfully run tests with your changes locally?



### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
Updated functions.php to include a new class called RegisterBlockType that simplifies registering new blocks. New package.json file. New event.php and event.js file.
